www/base/src/app/common/directives/buildsummary/buildsummary.tpl.jade

```diff
@@ -1,32 +1,36 @@
 .panel.panel-default(ng-class="results2class(buildsummary.build)")
   .panel-heading.no-select
-    .btn.btn-xs.btn-default(ng-click="buildsummary.toggleFullDisplay()", title="Expand all step logs")
-        i.fa.fa-chevron-circle-right.rotate(ng-class="{'fa-rotate-90':buildsummary.fulldisplay}")
-    .btn.btn-xs.btn-default(ng-click="buildsummary.toggleDetails()", title="Show steps according to their importance")
-        i.fa.fa-expand
-        {{buildsummary.levelOfDetails()}}
-    | &nbsp;
-    span(ng-if="buildsummary.prefix") {{buildsummary.prefix}}&nbsp;
-    a(ui-sref="build({builder:buildsummary.builder.builderid, build:buildsummary.build.number})")
-      | {{buildsummary.builder.name}}/{{buildsummary.build.number}}
-    .pull-right
-      span(ng-show="buildsummary.build.complete")
-          | {{(buildsummary.build.complete_at - buildsummary.build.started_at)| durationformat:'LLL' }}
-      span(ng-show="!buildsummary.build.complete")
-          | {{(buildsummary.now - buildsummary.build.started_at)| durationformat:'LLL' }}
-      span
-        | &nbsp;{{buildsummary.build.state_string}}&nbsp;
-      .label(ng-class="results2class(buildsummary.build)")
-        | {{results2text(buildsummary.build)}}
-      span(ng-if="buildsummary.parentbuild")
-          | &nbsp;
-          a.label(ng-class="results2class(buildsummary.parentbuild)",
-          ui-sref="build({builder:buildsummary.parentbuilder.builderid, build:buildsummary.parentbuild.number})")
-                | {{buildsummary.parentrelationship}}: 
-                | {{buildsummary.parentbuilder.name}}/{{buildsummary.parentbuild.number}}
+    .row
+      .col-sm-5
```

Comment:
I dont think splitting the UI like this is very efficient.
I would prefer sticking with

- stuff in the left
- stuff in the right

So that if the reason is long it can take space from the status string, and the other way around.